### ISSUE TITLE
Bug: `sort-classes`: Template literal expressions are not detected as dependencies 

### ISSUE THREAD

**Issue by hugop95** [azat-io/eslint-plugin-perfectionist#257]
### Describe the bug

Please see code below.

### Code example

```ts
class Class {
  b = ''
  a = `${this.b}`
}
```

This code is considered invalid when it should not.

### ESLint version

8.57.0

### ESLint Plugin Perfectionist version

v3.4.0

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Fixed in 4e0e6d8. Released in [v3.5.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.5.0).
